v4/integrations/nrgrpc/nrgrpc_client_test.go

```diff
@@ -103,7 +96,7 @@ func TestUnaryClientInterceptor(t *testing.T) {
 	if nil != err {
 		t.Fatal("cannot unmarshall client response", err)
 	}
-	if hdr, ok := hdrs["newrelic"]; !ok || len(hdr) != 1 || "" == hdr[0] {
+	if hdr, ok := hdrs["traceparent"]; !ok || len(hdr) != 1 || "" == hdr[0] {
```

Comment:
We no longer include the `newrelic` header. It is `traceparent` now instead.

Reply:
I see we use the constant `newrelic.DistributedTraceW3CTraceParentHeader` at other places. But probably the best is to stay as much in sync with `v3` as possible.